Setting up a local instance.md

```diff
@@ -0,0 +1,151 @@
+# Setting up a local instance of BackdropCMS.org
+
+This document describes how to set up a local instance of the BackdropCMS.org website using the sanitized databases (which are stripped of user accounts' email addresses and password hashes).
+
+BackdropCMS.org uses the CiviCRM module, which stores its data in a separate db. Because CiviCRM is a very complex module, it can be desirable to work on a version of the site without CiviCRM. We give instructions below for both with and without CiviCRM.
+
+## Checkout the repo from git
+
+Begin by checking out this repository, which we will assume creates a directory named `backdropcms.org` containing the repo somewhere on your local system.
+
+Set up your local webserver to point to `backdropcms.org/www/index.php`. 
+
+Do not run the Backdrop installer.
+
+## Download/install the sanitized databases and files
+
+There are sanitized nightly backups of the Backdrop database, CiviCRM database, and files directory (which includes config files), at https://sanitize.backdropcms.org. That site is password-protected; you will need to get the username and password from a Backdrop administrator.
+
+Once on that page, you will see backups for Backdropcms.org (first section), as well as for docs.backdropcms.org, forum.backdropcms.org, and events.backdropcms.org. Download the latest backups for backdropcms.org (first row), links to "Database", "CiviCRM", and "Files." The download files will have these names:
+
+* `backdropcmsorg-latest-sanitized.sql.gz`
+* `civicrm-latest-sanitized.sql.gz`
+* `backdropcmsorg-files-latest.tar.gz`
+
+`cd` to your download directory and unzip the first two (e.g., with `gunzip <filename>`) and expand the last one (e.g., with `tar -xyzf <filename>`).
+
+Remove the existing files directory at `backdropcms.org/www/files` and replace it with the files directory that you just unzipped.
+
+## Install config files
+
+The backdropcms.org site uses `backdropcms.org/config/live-active` as its config location, but for development, we will use `backdropcms.org/config/dev-active`. So,
+
+* `cd` to `backdropcms.org/config`
+* Remove all files in `dev-active`, e.g., with `sudo rm -r dev-active/*`.
+* Copy `live-active` to `dev-active`, e.g., with `cp live-active/* dev-active`.
+
+## Read in sanitized dbs
+
+In what follows, we assume the databases are named thus:
+
+* `backdropcmsorg` — for the Backdrop database
+* `civicrm` — for the CiviCRM database
+
+You may wish to name your local databases differently (for example, if you already have a `civicrm` database on your local MySQL). If so, you'll need to adjust the subsequent instructions accordingly.
+
+Read in the Backdrop database `backdropcmsorg-latest-sanitized.sql` and, if you will be using CiviCRM, read in the CiviCRM database `civicrm-latest-sanitized.sql`.
+
```

Comment:
Should we add a section on the database permissions that are needed? For backdrop I think we use
```
GRANT SELECT, INSERT, UPDATE, DELETE, CREATE, DROP, INDEX, ALTER,
CREATE TEMPORARY TABLES ON backdropcmsorg.*
TO 'username'@'127.0.0.1' IDENTIFIED BY 'password';
```
But for CIVI I think we need 
```
GRANT ALL ON civicrm.*
TO 'username'@'127.0.0.1' IDENTIFIED BY 'password';
```
I don't remember the specific civi permissions that were necessary, but if you know those it could be helpful to include them. This is my quick work-around to make sure things aren't broken :)

Reply:
Good point; I remember getting bitten once because Civi needed extra permissions (particularly related to foreign keys, IIRC). I'll add that.